src/components/collection-progress-group/docs.mdx

```diff
@@ -0,0 +1,63 @@
+---
+componentName: 'CollectionProgressGroup'
+---
+
+The `CollectionProgressGroup` component queries, parses, and displays collection progress information.
+
+It renders a `ButtonLink` to the "next tutorial" in the collection, as well as a `CollcetionProgressStatusSection`. The `CollectionProgressStatusSection` presents queried and parsed data. It can be used separately from `CollectionProgressGroup`.
+
+The examples below use `ButtonLink` and `CollectionProgressStatusSection`, which are the presentational components of `CollectionProgressGroup`.
+
+<LiveComponent>{`
+<div style={{ display: 'flex', gap: '16px' }}>
+  <ButtonLink
+    aria-label='Continue with some tutorial name.'
+    href='/some/tutorial'
+    text='Continue'
+  />
+  <CollectionProgressStatusSection
+    completedTutorialCount={4}
+    tutorialCount={12}
+  />
+</div>
+`}</LiveComponent>
+
+## Examples
+
+### CollectionProgressStatusSection
+
+<!-- Note: using LiveComponent here, as I couldn't get these examples
```

Comment:
I ran into some difficulties here in trying to render `ButtonLink` within `LiveComponent`, and `CollectionProgressStatusSection` outside of `LiveComponent`. For now have just worked around things, open to suggestions if there's a way to:
- Get `ButtonLink` available within `LiveComponent` without making it a named export of `components/collection-progress-group`
- Get `CollectionProgressStaticSection` available outside `LiveComponent`